deployment/ccip/changeset/aptos/cs_add_token_pool.go

```diff
@@ -31,13 +31,13 @@ type AddTokenPool struct{}
 func (cs AddTokenPool) VerifyPreconditions(env cldf.Environment, cfg config.AddTokenPoolConfig) error {
 	state, err := stateview.LoadOnchainState(env)
 	if err != nil {
-		return fmt.Errorf("failed to load existing Aptos onchain state: %w", err)
+		return fmt.Errorf("failed to load Aptos onchain state: %w", err)
 	}
 	var errs []error
 	// Validate supported chain
 	supportedChains := state.SupportedChains()
 	if _, ok := supportedChains[cfg.ChainSelector]; !ok {
-		errs = append(errs, fmt.Errorf("chain is not a supported: %d", cfg.ChainSelector))
```

Comment:
nit: Having the variable in the end of the string as `chain is not supported: %d` makes searching the error easier.